esmf-semantic-aspect-meta-model/src/main/resources/samm/meta-model/2.2.0/aspect-meta-model-shapes.ttl

```diff
@@ -1024,3 +1026,14 @@ samm:ConstrainListTypeUsageShape
          """
       ]
    ] .
+
+samm:ValueShape
+  a sh:NodeShape ;
+  sh:targetClass samm:Value ;
+  sh:property [
+    sh:path samm:value ;
+    sh:minCount 0 ;
```

Comment:
I think `sh:minCount` must also be 1. You can't declare a samm:Value without an actual value.